lib/video/postprocessing/visualmetrics/single/visualMetrics.js

```diff
@@ -9,7 +9,7 @@ const SCRIPT_PATH = require('../visualMetricsPath');
 
 module.exports = {
   async checkDependencies() {
-    return execa(SCRIPT_PATH, ['--check']);
+    return execa('python', [SCRIPT_PATH, '--check']);
```

Comment:
Break this into a separate changeset, and fold the `--perceptual` piece into the earlier commit.  We want to get this upstream, right?

Also, how about looking for `$PYTHON` in the environment, and then falling back to `python` on the `$PATH`?